Update docs (#171)
JuliaSmoothOptimizers/OptimizationProblems.jl: docs/src/benchmark.md

profile_solvers(stats, costs, costnames)
```

> It is also possible to select problems when initializing the problem list by filtering `OptimizationProblems.meta`.

Reviewer: ```suggestion
It is also possible to select problems when initializing the problem list by filtering `OptimizationProblems.meta`:
```

> Note that it is also possible to select the problems when initializing the problem list by searching into the `OptimizationProblems.meta`.

Reviewer: ```suggestion
It is also possible to select problems when initializing the problem list by filtering `OptimizationProblems.meta`.
```
